src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -242,7 +240,6 @@ public void GetIssues_NoIssuesOnServer_ReturnsEmptyList()
             SetupSolutionBinding(isBound: true, isConnected: true, projectKey: "sqkey", issues: null);
 
             var issuesProvider = new SonarQubeIssuesProvider(mockSqService.Object, mockTracker.Object, mockTimerFactory.Object);
-            VerifyServiceGetIssues(Times.Exactly(1)); // issues should be fetched on creation
```

Comment:
Again, the _VerifyServiceGetIssues_ check can't be called reliably unless we wait for the wait handle to signal, or until we have called _GetIssues_ (which will wait for the background task to complete). 

We don't need to check that the background task has started correctly in every test, so I've removed the check from this test and the following ones that are checking that the correct data is returned.